panel/lxqtpanelapplication.cpp

```diff
@@ -263,7 +263,7 @@ void LXQtPanelApplicationPrivate::loadBackend()
         }
     }
 
-    if ( preferredBackend.isEmpty() && xdgCurrentDesktops.contains( QStringLiteral("wlroots") ) )
+    if ( preferredBackend.isEmpty() && xdgSessionType == QStringLiteral("wayland") ) )
```

Comment:
The wlroots backend should be used only if `wlroots` is included. That inclusion means, "we know that it works with this Wayland compositor." The dummy backed should be used with unsupported compositors.